src/views/settings/ExternalAccount/IzlyActivation.tsx

```diff
@@ -1,14 +1,14 @@
-import React, {useState, useEffect} from "react";
-import type {Screen} from "@/router/helpers/types";
-import {useTheme} from "@react-navigation/native";
-import {SafeAreaView, useSafeAreaInsets} from "react-native-safe-area-context";
-import { Alert, Keyboard, KeyboardAvoidingView, StyleSheet, TouchableWithoutFeedback, View} from "react-native";
+import React, { useState, useEffect } from "react";
+import type { Screen } from "@/router/helpers/types";
+import { useTheme } from "@react-navigation/native";
+import { SafeAreaView, useSafeAreaInsets } from "react-native-safe-area-context";
+import { Alert, Keyboard, KeyboardAvoidingView, StyleSheet, TouchableWithoutFeedback, View } from "react-native";
 import PapillonShineBubble from "@/components/FirstInstallation/PapillonShineBubble";
-import { NativeText,} from "@/components/Global/NativeComponents";
+import { NativeText, } from "@/components/Global/NativeComponents";

```

Comment:
```suggestion
import { NativeText } from "@/components/Global/NativeComponents";
```
Petite virgule en trop